src/proof/supernova.rs

```diff
@@ -231,14 +231,14 @@ where
         pp: &Self::PublicParams,
         z0: &[F],
         zi: &[F],
-        last_circuit_idx: usize,
+        _last_circuit_idx: usize,
```

Comment:
If this is no longer necessary, we should open an issue about it.

Reply:
I'd vote for removing the unused argument from the API, and fixing downstream (lurk-rs) as part of integration of this work. That could possibly be by adjusting https://github.com/lurk-lab/lurk-rs/pull/936.

Reply:
Whoops… I thought I was looking at an arecibo change.

Reply:
We should definitely remove rather than ignore vestigial arguments — unless I'm missing something.

Reply:
Parity with Nova, I guess…

Reply:
`verify` is an unified API now. We need to pass at least something, maybe `PhantomData`. We define the extra input for `verify` when implementing `RecursiveSNARKTrait`, which is currently `usize` for both Nova and SuperNova proofs.